### DISCUSSION TITLE
Invisible blocks

### DISCUSSION THREAD

**Discussion by Giehek** [MrTJP/ProjectRed#1668] [Q&A] [answered]
I found a bug with invisible blocks. F3 + T doesn't fix it.
Someone know how to fix this problem?

![2021-08-10_21 14 49](https://user-images.githubusercontent.com/63113462/128921357-5e90b269-8237-4e55-801d-ac1af58052f6.png)

**Comment by MrTJP** [comment]
I need some info first. What are the versions of everything? What other mods are installed? Any texture packs?

**Reply by Giehek** [reply]
It's version 1.16.5 - 4.10.0. I don't use texture packs and there is a lot of mods

**Reply by Giehek** [reply]
I uninstalled Soudium mod and now everything works.

**Reply by Giehek** [reply]
Probbably Sodium and mc multipart don't cooperate

**Comment by MrTJP** [comment, marked as answer]
Seems to involve Sodium. If this happens with any Multipart block, then probably something incompatible between Sodium and CBMultipart.

**Reply by nikidigiano** [reply]
I have the same issue but Sodium is not installed.

**Comment by MrTJP** [comment]
Closing this discussion now. This issue has come up and been resolved a few times since this was posted, and almost always is related to rendering/performance tweaks such as Sodium, Rubidium, Optifine, etc.
